example/android/app/build.gradle

```diff
@@ -76,7 +76,9 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "index.js"
+    entryFile: "example/index.js",
+    // TODO: remove once @esemesek's patch is merged
+    cliPath: "../node_modules/react-native/cli.js"
```

Comment:
cc @Esemesek @grabbou we had to change entryFile so that Metro can find it. Ideas how we can mitigate that?